sauna/plugins/ext/supervisor.py

```diff
@@ -0,0 +1,166 @@
+from sauna.plugins import Plugin, PluginRegister
+import xmlrpc.client
+import http.client
```

Comment:
I usually try to separate imports in groups: from stdlib, external dependencies and imports from the project. Like:

```
import socket
import re

import requests

from . import foo
```